### PR TITLE
Remove stretch support

### PR DESCRIPTION
**Pull request by behrmann**

### CODE REVIEW

File: mkosi/distributions/debian.py [hunk 1]
```diff
@@ -124,7 +115,6 @@ def install_packages(cls, state: MkosiState, packages: Sequence[str], apivfs: bo
         setup_apt(state, cls.repositories(state))
         invoke_apt(state, "update", apivfs=False)
         invoke_apt(state, "install", packages, apivfs=apivfs)
-        install_apt_sources(state, cls.repositories(state, local=False))
```
Comment (bluca) [inline mkosi/distributions/debian.py:127]: Please leave this, it is useful to have, and if someone wants to build a really minimal one they can remove the package manager and then it won't be added

Comment (behrmann) [reply mkosi/distributions/debian.py]: Dropped the commit from this PR and spin it out into a separate one after testing this again. I remember it not doing anything in the first place and parity between what we do for different distros would be nice, but this definitely needs more discussion.

**Comment by DaanDeMeyer** [comment]
cc @gdiscry are you OK with maintaining the stretch support in a fork?

**Comment by gdiscry** [comment]
> cc @gdiscry are you OK with maintaining the stretch support in a fork?

Yes, no problem, I can maintain a fork for my needs. And the fix is "documented" in the project history.

With a bit of luck, I won't need it for too long.

**Review by bluca — APPROVED** [review]

**Pull request merged**